example/src/App.tsx

```diff
@@ -36,117 +36,85 @@ const styles = StyleSheet.create({
     borderRadius: 8,
     marginVertical: 8,
     padding: 12,
-    fontStyle: "normal",
-    fontWeight: "normal",
+    fontStyle: 'normal',
+    fontWeight: 'normal',
     borderWidth: 1,
-    width: "100%",
+    width: '100%',
   },
-})
+});
 
 export default function App() {
+  const [baseUrl, setBaseUrl] = useState<string>(
+    process.env.EXPO_PUBLIC_BASE_URL || ''
+  );
 
-  const [roomUrl, setRoomUrl] = useState<string>(process.env.EXPO_PUBLIC_BASE_URL || '')
+  const [pipecatClient, setPipecatClient] = useState<RTVIClient | undefined>();
 
-  const [pipecatClient, setPipecatClient] = useState<RTVIClient|undefined>()
-
-  const [inCall, setInCall] = useState<boolean>(false)
-  const [currentState, setCurrentState] = useState<TransportState>("disconnected")
+  const [inCall, setInCall] = useState<boolean>(false);
+  const [currentState, setCurrentState] =
+    useState<TransportState>('disconnected');
 
   const createPipecatClient = () => {
     return new RTVIClient({
       transport: new RNDailyTransport(),
       params: {
-        baseUrl: roomUrl,
-        config: [
```

Comment:
wait so does this example project now need a server-side component for it to work? i don't see it added in this pr

Reply:
It is not. Right now I am using the same server side provided by the `simple-chatbot` example for testing. I could add it here.

Reply:
or is the expectation that folks already know how to run a pipecat pipeline with rtvi integration?

wonder if we should do a bit more hand-holding, at least with something in the README.

we also need to update env.example to not point to daily bots, right?

Reply:
ah sorry didn't see your message come in as i was writing my next one

Reply:
My expectation for this example is that it's for internal use only, so we can validate the changes within the transport before making a new release.

For those only using `@pipecat-ai/react-native-daily-transport`, we'll provide an example within Pipecat itself, which I’m including in this PR:
- https://github.com/pipecat-ai/pipecat/pull/1746

That’s why I’m not sure we should include the server-side code here.

Reply:
+1 to not including the full server-side pipecat code in this repo. a README + env.example update would be sufficient

Reply:
Ok, I have changed both. Let me know what do you think. 🙂